### PR TITLE
VACMS-2533 fix paragraph test and reformat all tests

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #2533 
This pr adds a patch to the TableEqualityAssertion class required by spec tool. At present, when duplicate entries are detected in a .feature file, an ambiguous / unhelpful error is thrown. This pr outputs a message indicating a duplicate has been detected, and the duplicate line.

This pr also reformats dst .feature files to match the formatting in the spec tool google sheet.

Depends on https://github.com/department-of-veterans-affairs/va.gov-cms/pull/2567

## Testing done
Behat

## Screenshots
image

## QA steps
Add a duplicate line to any dst feature file and run the test - visually verify that the duplicate line message and line appear in the error message.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -308,6 +308,9 @@
             },
             "drupal/workflow_participants": {
                 "Fix revision uncaught type error by extending class.": "https://www.drupal.org/files/issues/2019-08-16/workflow_participants-revision-error-3075426-1.patch"
+            },
+            "traviscarden/behat-table-comparison": {
+                "Throw meaningful message for duplicate entry.": "https://patch-diff.githubusercontent.com/raw/TravisCarden/behat-table-comparison/pull/7.patch"
```
Comment (swirtSJW) [inline composer.json:313]: I think if we are specifically adding this patch, then it may be worth explicitly requiring the package.   As in 
`lando composer require traviscarden/behat-table-comparison`

File: composer.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
         "drupal-composer/drupal-scaffold": "^2.0.0",
         "drupal/address": "^1.4",
         "drupal/admin_toolbar": "^1.24",
-        "drupal/auto_entitylabel": "^2.1@beta",
+        "drupal/auto_entitylabel": "3.0-beta3",
```
Comment (swirtSJW) [inline composer.json:22]:
```suggestion
        "drupal/auto_entitylabel": "^3.0-beta3",
```
We want this to be able to float upward from here.

Comment (swirtSJW) [reply composer.json:22]: Lets leave as is. will fix on thenext  PR
